### ISSUE TITLE
Accessibility: Unable to access "Apply Filters" and "Clear All" buttons on dashboard filters using keyboard navigation 

### ISSUE THREAD

**Issue by yousoph** [apache/superset#25860]
When using the tab key to navigation a dashboard, the user is unable to access the Apply Filters button and the Clear All button on the filter bar

#### How to reproduce the bug

1. Create a dashboard with filters 
2. Used tabs to navigation the dashboard 

### Expected results
User should be able to access the Apply Filter and Clear All buttons on the filter bar with keyboard navigation
https://www.w3.org/TR/WCAG22/#keyboard

### Actual results

The Apply Filter and Clear All buttons are skipped when navigating using the tab key
* Icons, check boxes, radio buttons should have a 2px border radius, other items should have a 4px border radius 
* Order of the tabbing: top navigation, dashboard title bar, filters (horizontal or vertical), tab bar, dashboard content

### Environment

- browser type and version:
- superset version: `master`
- python version: `python --version`
- node.js version: `node -v`
- any feature flags active:

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by yousoph** [comment]
https://github.com/apache/superset/pull/26138
